api/services/control/control.proto

```diff
@@ -163,3 +164,14 @@ message InfoRequest {}
 message InfoResponse {
 	moby.buildkit.v1.types.BuildkitVersion buildkitVersion = 1;
 }
+
+message SourcePolicy {
+	repeated BuildInfoSource Sources = 1;
+}
+
+message BuildInfoSource {
+	string Type = 1;
+	string Ref = 2;
+	string Alias = 3;
```

Comment:
Alias does not have a definition in here I believe.

I think rather than buildinfo this structure is more similar to the named contexts https://github.com/moby/buildkit/pull/2521 feature, except this time implemented in the LLB level and applying to all builds that are part of the same user request.

So it shouldn't be only possible to set the digest(pin) for the same source, but for example map "alpine:latest" to "busybox:latest", image ref from a registry to a local `oci-layout://` directory. Replace Git URL with HTTP URL etc. Maybe also replace a source with another LLB definition like linked targets in `buildx bake` do (not sure how that would be defined though).

Reply:
Removed `Alias`.
I guess we will add back this as "Replace" later (in another PR)
cc @cpuguy83